src/i18n/locales/en.json

```diff
@@ -400,6 +400,8 @@
         "audioExclusiveMode_description": "enable exclusive output mode. In this mode, the system is usually locked out, and only mpv will be able to output audio",
         "audioPlayer": "audio player",
         "audioPlayer_description": "select the audio player to use for playback",
+        "buttonSize": "Button size",
+        "buttonSize_description": "The size of the player bar buttons",
```

Comment:
If the units are not listed in the number input, they should probably be listed here

Reply:
Added them to the input.